src/library-authoring/LibraryContent.test.tsx

```diff
@@ -56,7 +57,11 @@ const clipboardBroadcastChannelMock = {
 
 const withLibraryId = (libraryId: string) => ({
   extraWrapper: ({ children }: { children: React.ReactNode }) => (
-    <LibraryProvider libraryId={libraryId}>{children}</LibraryProvider>
+    <LibraryProvider libraryId={libraryId}>
+      <SidebarProvider>
+        {children}
+      </SidebarProvider>
```

Comment:
@rpenido Now that we have default values for sidebar context, the providers can be removed from tests that don't actually use it. For example, this one works without `SidebarProvider`. Similarly we can simplify other tests, which was the reason for adding default values.

Reply:
Thanks @navinkarkera! Fixed!